src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/commands.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#pylint: disable=line-too-long
+
+from azure.cli.core.commands import cli_command
+import azure.cli.command_modules.documentdb
+from azure.cli.command_modules.documentdb._client_factory import (cf_documentdb)
+
+cli_command(__name__, 'documentdb show', 'azure.cli.command_modules.documentdb.sdk.operations.database_accounts_operations#DatabaseAccountsOperations.get', cf_documentdb)
```

Comment:
It would also be better to extract out the base string to a variable like so:
```Python
custom_path = 'azure.cli.command_modules.documentdb.sdk.operations.database_accounts_operations#{}'

cli_command(__name__, 'documentdb list', custom_path.format('DatabaseAccountsOperations.list_by_resource_group'), cf_documentdb)
```